src/main/java/at/hannibal2/skyhanni/events/ItemClickEvent.kt

```diff
@@ -0,0 +1,8 @@
+package at.hannibal2.skyhanni.events
+
+import at.hannibal2.skyhanni.data.ClickType
+import at.hannibal2.skyhanni.utils.BlockUtils.getBlockStateAt
+import at.hannibal2.skyhanni.utils.LorenzVec
+import net.minecraft.item.ItemStack
+
+class ItemClickEvent(val itemInHand: ItemStack?) : LorenzEvent()
```

Comment:
Naming can be confusing since BlockClickEvent already handles left or right-clicking.